mantle/platform/conf/conf.go

```diff
@@ -379,19 +379,20 @@ func (c *Conf) MergeV34exp(newConfig v34exptypes.Config) {
 
 // Merge all configs into a V3.4 config
 // configs must be of type V34exp to merge
-func MergeAllV34exp(confObjs []*Conf) (*UserData, error) {
+func MergeAllConfigs(userdataObjs []*UserData) (*UserData, error) {
 	config := Conf{}
-	config.ignitionV34exp = &v34exptypes.Config{
-		Ignition: v34exptypes.Ignition{
-			Version: "3.4.0-experimental",
+	config.ignitionV33 = &v33types.Config{
+		Ignition: v33types.Ignition{
+			Version: "3.3.0",
 		},
 	}
-	for _, conf := range confObjs {
-		if conf.ignitionV34exp == nil {
-			return nil, fmt.Errorf("configs must be V34exp to merge")
+	objectsToMerge := &config.ignitionV33.Ignition.Config.Merge
+	for _, conf := range userdataObjs {
+		url := dataurl.EncodeBytes([]byte(conf.data))
```

Comment:
This means we're always treating the individual configs as opaque blobs, so we no longer catch validation errors before the config is handed to the machine.  I think there's value in round-tripping the config through the `Render()` and `String()` methods before encoding, so all configs go through the same validation path.  To separate out error handling, you might have this function continue taking `[]*Conf`, and do the `Render()`s in the caller.